airflow/models/mappedoperator.py

```diff
@@ -196,6 +196,9 @@ class MappedOperator(AbstractOperator):
     upstream_task_ids: Set[str] = attr.ib(factory=set, init=False)
     downstream_task_ids: Set[str] = attr.ib(factory=set, init=False)
 
+    inlets: list = attr.ib(factory=list)
+    outlets: list = attr.ib(factory=list)
```

Comment:
I think these should be properties. Also probably don’t need to be declared in AbstractOperator?

Reply:
Properties looking at where? `parital_kwargs`?

Reply:
Yeah and populate the partial_kwargs key in `BaseOperator.partial` and `_TaskDecorator.map`.